05week/bankAccount.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+class BankAccount {
+  constructor(accountNumber, owner) {
+    this.accountNumber = accountNumber;
+    this.owner = owner;
+      //this is supposed to be the transaction history 
+    this.transactions = [];
+  }
+
+  //~~isn't returning the correct values 
+  balance() {
+    let balance = 0
+    console.log('inside the balane?')
```

Comment:
for this to work, you would need to map each transaction in the array to an amount and then call reduce over the amounts.